api/omnibox/simple-example/background.js

```diff
@@ -0,0 +1,21 @@
+// Copyright 2021 Google LLC
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file or at
+// https://developers.google.com/open-source/licenses/bsd
+
+// This event is fired each time the user updates the text in the omnibox,
+// as long as the extension's keyword mode is still active.
+chrome.omnibox.onInputChanged.addListener((text, suggest) => {
+  console.log('inputChanged: ' + text);
+  suggest([
```

Comment:
I assume you are working on this since it is an open PR but this line triggers an error when I run this example

```
Error in event handler: ReferenceError: DOMParser is not defined
    at parseOmniboxDescription (extensions::omnibox:24:19)
    at extensions::omnibox:101:25
    at suggestCallback (extensions::omnibox:116:20)
    at chrome-extension://eieflmjhclgndndbllffkimmokpfhibl/background.js:11:3
    at extensions::omnibox:118:3
```

Reply:
Yep, I may end up splitting this PR into 2 as one of the samples works and one does not and I don't want to block on merging this PR until both are functional.